client/src/test/java/com/vesoft/nebula/encoder/TestEncoder.java

```diff
@@ -62,8 +76,39 @@
         dateValue.setDVal(new Date((short)2020, (byte)2, (byte)20));
         final Value nullVal = new Value();
         nullVal.setNVal(NullType.__NULL__);
+        // geograph point
+        // POINT(179.0 89.9)
+        final Value geogPointVal = new Value();
+        geogPointVal.setGgVal(Geography.ptVal(new Point(new Coordinate(179.0, 89.9))));
+        // geography linestring
+        // LINESTRING(0 1, 1 2, 3 7)
+        final Value geogLineStringVal = new Value();
+        List<Coordinate> line = new ArrayList<>();
+        line.add(new Coordinate(0, 1));
+        line.add(new Coordinate(1, 2));
+        line.add(new Coordinate(3, 7));
+        geogLineStringVal.setGgVal(Geography.lsVal(new LineString(line)));
+        // geography polygon
+        // POLYGON((-108.7 35.0, -100.0 46.5, -90.7 34.9, -108.7 35.0),
+        // (-100.1 41.4, -102.9 37.6, -96.8 37.5, -100.1 41.4))
+        final Value geogPolygonVal = new Value();
+        List<Coordinate> shell = new ArrayList<>();
+        shell.add(new Coordinate(-108.7, 35.0));
+        shell.add(new Coordinate(-100.0, 46.5));
+        shell.add(new Coordinate(-90.7, 34.9));
+        shell.add(new Coordinate(-108.7, 35.0));
+        List<Coordinate> hole = new ArrayList<>();
+        hole.add(new Coordinate(-100.1, 41.4));
+        hole.add(new Coordinate(-102.9, 37.6));
+        hole.add(new Coordinate(-96.8, 37.5));
+        hole.add(new Coordinate(-100.1, 41.4));
+        List<List<Coordinate>> rings = new ArrayList<List<Coordinate>>();
+        rings.add(shell);
+        rings.add(hole);
+        geogPolygonVal.setGgVal(Geography.pgVal(new Polygon(rings)));
```

Comment:
please add test for null geograph value.

Reply:
It's not a need, because null geo value is no different from other types. And TestEncoder.java also just test null value of int64

Reply:
It's necessary, it happens `out of bound error` before for null string value int the properties. So you should add test like `there are three geography properties but one is null` to avoid that problem.

Reply:
OK